src/styles/colors.js

```diff
@@ -43,6 +43,33 @@ const darkModeColors = {
 
 const isHex = (color = '') => color.length >= 3 && color.charAt(0) === '#';
 const isRGB = (color = '') => toLower(color).substring(0, 3) === 'rgb';
+
+const avatarBackgrounds = [
```

Comment:
sooo I think...
Can we inline those colors to `src/utils/defaultProfileUtils.ts`? Then everywhere we rely on `avatarBackgrounds[i]` just use `avatars[i].color` from `src/utils/defaultProfileUtils.ts`?
| just want to omit the mapping between indices in two places.

Reply:
I see what you're saying, but the problem is that `avatars[i].color` will not be a list of unique colors since some of the avatars share the same background color as you can see in `defaultProfileUtils.js`. We still need a list of just the new colors that we can use in the same way we were using `colors.avatarColor`. Let me know if you think there's a better way around this!